src/pages/documentation/adobe-target/tabs/api-reference.md

```diff
@@ -52,7 +52,228 @@ clearPrefetchCache();
 ACPTarget.clearPrefetchCache();
 ```
 
-<Variant platform="android" api="clicked-location" repeat="6"/>
+<Variant platform="android" api="extension-version" repeat="5"/>
+
+#### Java
+
+**Syntax**
+
+```java
+public String extensionVersion()
+```
+
+**Example**
+
+```java
+Target.extensionVersion();
+```
+
+<Variant platform="ios" api="extension-version" repeat="7"/>
+
+**Syntax**
+
+```objc
++ (nonnull NSString*) extensionVersion;
+```
+
+**Example**
+
+**Swift** 
+
+```swift
+let targetVersion = ACPTarget.extensionVersion()
+```
+
+**Objective-C**
+
+```objc
+NSString *targetVersion = [ACPTarget extensionVersion];
+```
+
+<Variant platform="react-native" api="extension-version" repeat="5"/>
+
+#### JavaScript
+
+**Syntax**
+
+```javascript
+extensionVersion(): Promise<string>
+```
+
+**Example**
+
+```javascript
+ACPTarget.extensionVersion().then(version => {
+            // read Target extension version 
+});
+```
+
+<Variant platform="android" api="get-session-id" repeat="6"/>
+
+#### Java
+
+**Syntax**
+
+```java
+public static void getSessionId(final AdobeCallback<String> callback)
+```
+
+* _callback_ is invoked with the `sessionId` value, or `null` if there was an error retrieving it.
+
+**Example**
+
+```java
+Target.getSessionId(new AdobeCallback<String>() {                    
+    @Override
+    public void call(String sessionId) {
+                // read Target sessionId
+    }
+});
+```
+
+<Variant platform="android" api="get-third-party-id" repeat="6"/>
+
+#### Java
+
+**Syntax**
+
+```java
+public static void getThirdPartyId(final AdobeCallback<String> callback)
+```
+
+* _callback_ is invoked with the `thirdPartyId` value. If no third-party ID was set, this value will be `null`.
+
+**Example**
+
+```java
+Target.getThirdPartyId(new AdobeCallback<String>() {                    
+    @Override
+    public void call(String thirdPartyId) {
+                // read Target thirdPartyId
+    }
+});
+```
+
+<Variant platform="ios" api="get-third-party-id" repeat="8"/>
+
+**Syntax**
+
+```objc
++ (void) getThirdPartyId: (nonnull void (^) (NSString* __nullable thirdPartyId)) callback;
+```
+
+* _callback_ is invoked with the `thirdPartyId` value. If no third-party ID was set, this value will be `nil`.
+
+**Example**
+
+**Swift**
+
+```swift
+ACPTarget.getThirdPartyId({thirdPartyID in
+       // read Target thirdPartyId
+})
+```
+
+**Objective-C**
+
+```objc
+[ACPTarget getThirdPartyId:^(NSString *thirdPartyId){
+       // read Target thirdPartyId
+}];
+```
```

Comment:
It's not super important, since this is previous versions, but don't we also want to do it like

#### Language

**Syntax**

**Example**

Reply:
Yeah I updated it for the latest version and didn't do it for the previous version as it's not super important at this point.